src/app/static/src/tests/integration/adr-dataset.itest.ts

```diff
@@ -98,7 +98,7 @@ describe("ADR dataset-related actions", () => {
         expect(commit.mock.calls[2][0].type).toStrictEqual(ADRMutation.SetDatasets);
         const datasets = commit.mock.calls[2][0]["payload"];
         const dataset = datasets.find((dataset: any) => dataset.organization.name === "naomi-development-team");
-        expect(dataset).not.toBeNull()
+        expect(dataset).not.toBeUndefined();
```

Comment:
This assertion was previously incorrect - `find()` returned `undefined` if not found, not `null`